### PR TITLE
add vscode folder ;; WLFE-68

### LINKED ISSUE #40: Add a "vscode" folder to the web application template
As our default IDE is VSCode, template should include a default `.vscode` folder.

The folder should include default settings to:
- Sort imports
- Lint code
- Format code on save
- Exclude `dist` and `node_modules` folder from search
- `"typescript.tsdk": "node_modules\\typescript\\lib"`

The folder should also include default VSCode extensions to install.

### PR DESCRIPTION
**Pull request by alexasselin008**
Closes #40

### CODE REVIEW

File: templates/web-application/.vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    "javascript.preferences.useAliasesForRenames": false,
+    "typescript.preferences.useAliasesForRenames": false,
+    "json.format.enable": false,
+    "css.validate": false,
+    "eslint.packageManager": "pnpm",
+    "eslint.validate": ["javascript", "javascriptreact", "mdx"],
+    "stylelint.autoFixOnSave": true,
+    "stylelint.useLocal": true,
+    "files.associations": {
+        "*.snap": "javascriptreact"
```
Comment (patricklafrance) [inline templates/web-application/.vscode/settings.json:23]: What are `*.snap` files?

File: templates/web-application/.vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        "editor.formatOnSave": false
+    },
+    "javascript.format.enable": false,
+    "javascript.validate.enable": true,
+    "javascript.preferences.useAliasesForRenames": false,
+    "typescript.preferences.useAliasesForRenames": false,
+    "json.format.enable": false,
+    "css.validate": false,
+    "eslint.packageManager": "pnpm",
+    "eslint.validate": ["javascript", "javascriptreact", "mdx"],
```
[the remark this answers is not in the document]
Comment (alexasselin008) [reply templates/web-application/.vscode/settings.json:19]: Nope

[the remark this answers is not in the document]
Comment (alexasselin008) [reply templates/web-application/.vscode/settings.json:19]: Only the SDK location

[the remark this answers is not in the document]
Comment (alexasselin008) [reply templates/web-application/.vscode/settings.json:19]: According to their documentation, we could remove this line: 
> eslint.validate - an array of language identifiers specifying the files for which validation is to be enforced. This is an old legacy setting and should in normal cases not be necessary anymore. Defaults to ["javascript", "javascriptreact"].

https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint

File: templates/web-application/.vscode/settings.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    "stylelint.useLocal": true,
+    "files.associations": {
+        "*.snap": "javascriptreact"
+    },
+    "breadcrumbs.enabled": true,
+    "search.exclude": {
+        "**/node_modules*": true,
+        "**/dist*": true
+    },
+    "timeline.showView": true,
```
Comment (alexasselin008) [inline templates/web-application/.vscode/settings.json:30]: timeline.showView seems deprecated

File: templates/web-application/.vscode/settings.json [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+        "source.organizeImports": false
+    },
+    "editor.formatOnSave": true,
+    "[html]": {
+        "editor.formatOnSave": false
+    },
+    "javascript.format.enable": false,
+    "javascript.validate.enable": true,
+    "javascript.preferences.useAliasesForRenames": false,
+    "typescript.preferences.useAliasesForRenames": false,
```
[the remark this answers is not in the document]
Comment (patricklafrance) [reply templates/web-application/.vscode/settings.json]: Oh boy, thank you for this setting 🙏🏻

**Review by ofrogon — APPROVED** [review]

**Pull request merged**